src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  const married = isMarried
+    ? `${partnerName} is my ${partner}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
hard to read, why write everything in one line?

If you think it's cool... no... forget it... the number of lines in your file never correlates with the quality of your code - usually after refactoring, when you make the code readable, the number of lines increases